### DISCUSSION TITLE
请问最近r7800等IPQ806X编译是不是没有nss驱动了？

### DISCUSSION THREAD

**Discussion by chris7395328** [coolsnowwolf/lede#11886] [Q&A / 快速问答]
rt。很久没编译了发现没有nss了

**Comment by bilibilistack** [comment]
同样有这个问题，怎么开启呢

**Comment by bilibilistack** [comment]
内核1月份升级5.10了，应该是暂时不兼容了
https://github.com/coolsnowwolf/lede/commit/7515f84eaedc7b2b190eefddf8abbfd04685ec49
